02-Git/Game.c

```diff
@@ -24,7 +24,7 @@ int main()
 
     int user_wins = 0;
     int comp_wins = 0;
-    
+
```

Comment:
Do this in commit where these trailing spaces were added

Reply:
Fixed